.github/workflows/build-docker.yaml

```diff
@@ -8,7 +8,7 @@ on:
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository }}
-
+  GOPRIVATE: "github.com/speakeasy-api"
```

Comment:
So this represents a significant change.

Before this our whole action is opensource, can be forked and modified as need but this makes it not possible without a big rewrite for someone

Reply:
can we do this without pulling in a private dependency

Reply:
done.